Makefile

```diff
@@ -97,7 +98,18 @@ deploy:
 		--set="image.name=ko://github.com/shipwright-io/triggers" \
 		shipwright-triggers \
 		$(CHART_DIR) | \
-			ko apply $(KO_OPTS) $(ARGS) --filename -
+			KO_DOCKER_REPO="$(IMAGE_HOST)/$(IMAGE_NAMESPACE)" GOFLAGS="${GOFLAGS} ko apply $(KO_OPTS) $(ARGS) --filename -
+
+# uses helm to create a package and ko for the container image
+.PHONY: release
+release: CGO_ENABLED=0
+release:
+	helm template \
+		--namespace=$(NAMESPACE) \
+		--set="image.name=ko://github.com/shipwright-io/triggers" \
+		shipwright-triggers \
+		$(CHART_DIR)| \
```

Comment:
Let export `helm template` as a reusable macro, as in:

```Makefile
# uses helm to render the resource templates to the stdout
define render-template
    @helm template \
		--namespace=$(NAMESPACE) \
		--set="image.name=ko://github.com/shipwright-io/triggers" \
		shipwright-triggers \
		$(CHART_DIR)
endef
```
Having it defined, you can use on the `deploy` and `release` targets as the following snippet:

```Makefile
.PHONY: release
release: CGO_ENABLED=0
release:
    $(call render-template) | \
        ko resolve $(KO_OPTS) $(ARGS) \
            --platform=all \
            --recursive \
            --filename - >release.yaml
```

Also, make sure you're trying out the targets on your workstation, please. The `release` target contains a non-close quote for the `GOFLAGS` inline variable.

Reply:
fixed